Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -493,6 +507,42 @@ static void __GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameS
                 line->_width += glyphRunDetails._dwriteGlyphRun[i].glyphAdvances[index];
             }
 
+            DWRITE_GLYPH_METRICS glyphMetrics[run->_dwriteGlyphRun.glyphCount];
```

Comment:
They'll be used when determining the frame size when creating a frame as well, and we don't really have .